src/main/test/com/epam/preprod/sirenko/wrappers/FileReaderWrapperTest.java

```diff
@@ -0,0 +1,67 @@
+package com.epam.preprod.sirenko.wrappers;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FileReaderWrapperTest {
+	File file;
+	BufferedWriter bufferedWriter;
+	
+	@BeforeEach
+	public void setData() throws IOException {
+		file = new File("test.txt");
```

Comment:
We can divide it to 2 private methods setupFile and setupReader for better readability